.github/workflows/integrations-ghcr.yml

```
@@ -55,7 +55,7 @@ jobs:
   build-publish:
     name: Build ${{ matrix.name }}
     runs-on: ubuntu-latest
-    if: ${{ github.event_name != 'workflow_dispatch' || inputs.integration == 'all' }}
+    if: ${{ github.event_name != 'workflow_dispatch' || inputs.integration == 'all' || inputs.integration == matrix.name }}
```

actionlint (1.7.9)

context "matrix" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)